app/app.go

```diff
@@ -252,15 +252,15 @@ type App struct {
 	ScopedTransferKeeper capabilitykeeper.ScopedKeeper
 	ScopedWasmKeeper     capabilitykeeper.ScopedKeeper
 
-	aolKeeper    aolkeeper.Keeper
-	didKeeper    didkeeper.Keeper
-	burnKeeper   burnkeeper.Keeper
-	tokenKeeper  tokenkeeper.Keeper
-	wasmKeeper   wasm.Keeper
-	marketKeeper marketkeeper.Keeper
+	aolKeeper      aolkeeper.Keeper
+	didKeeper      didkeeper.Keeper
+	burnKeeper     burnkeeper.Keeper
+	tokenKeeper    tokenkeeper.Keeper
+	wasmKeeper     wasm.Keeper
+	datadealKeeper datadealkeeper.Keeper
```

Comment:
How about changing the field names declared in the struct as follows according to camel expression?
```suggestion
	dataDealKeeper datadealkeeper.Keeper
```

Reply:
I thought that `datadeal` has been used as a single word intentionally.
Do you think it is better to declared with camel expression?

Reply:
I don't think that's wrong either.

Well, our URL notation looks like `data-deal`, `data-pool`.(https://github.com/medibloc/panacea-ecosystem-arch/blob/main/data-marketplace/data-pool-model/NFT-model.md#downloading-data)

Looking at the notation above, I think it would be better to write the two words separately.

Reply:
I got it. Thanks for kind comments.
Then, it would be better to use separate words in module name as well, `data-deal`. wdyt?

Reply:
Is the module name you are talking about this link?
https://github.com/medibloc/panacea-core/tree/ft/273/rename-module/x/datadeal

Reply:
Yes right. 
For now, however, I think we use camel expression for variables and `x/datadeal` for module name as we discussed before.